core/blockly.js

```diff
@@ -300,20 +300,14 @@ Blockly.hideChaff = function(opt_allowToolbox) {
   Blockly.Tooltip.hide();
   Blockly.WidgetDiv.hide();
   Blockly.DropDownDiv.hideWithoutAnimation();
+
   if (!opt_allowToolbox) {
```

Comment:
I just noticed this was a parameter in hideChaff, which introduces some complication.
I'm not sure how to document that autoCloseables are only hidden it this is false. The documentation for this parameter isn't entirely accurate either because it ALSO prevents the trashcan from automatically closing.

Reply:
Updated to pass parameter through and jsDoc updated.